wallet/src/main/java/com/android/identity_credential/wallet/MainActivity.kt

```diff
@@ -0,0 +1,477 @@
+@file:OptIn(ExperimentalMaterial3Api::class, ExperimentalFoundationApi::class)
```

Comment:
add the copyright here

Reply:
I'm actually noticing now that the appholder/appverifier MainActivity classes also don't have the copyright - should we add to these types of files or leave off?

Reply:
I'm actually not sure what to put in the copyright header (or if it's even advisable to have one) so I suggest we leave that out for now until I consult with OWF.